api/web3server.go

```diff
@@ -373,17 +373,13 @@ func (svr *Web3Server) getTransactionCount(in interface{}) (interface{}, error)
 }
 
 func (svr *Web3Server) call(in interface{}) (interface{}, error) {
-	from, to, gasLimit, value, data, err := parseCallObject(in)
+	callerAddr, to, gasLimit, value, data, err := parseCallObject(in)
 	if err != nil {
 		return nil, err
 	}
 	if to == "io1k8uw2hrlvnfq8s2qpwwc24ws2ru54heenx8chr" {
 		return nil, nil
 	}
```

Comment:
why check "io1k8uw2hrlvnfq8s2qpwwc24ws2ru54heenx8chr"?

Reply:
pls add a comment that this is the special balance checker contract (which metamask will check periodically)